docs/_getting-started/index.md

```diff
@@ -1,103 +1,43 @@
 ---
 title: Getting Started with Geth
-permalink: getting-started
+permalink: docs/getting-started
+sort_key: A
 ---
 
-## Installing
+To use Geth, you need to install it first. You can install the geth software in a variety
```

Comment:
Hmm, not sure about this change, I did spend some time adding these sections to the getting started guide so people didn't need to switch back and forth between pages. Also the instructions here don't tell everyone everything, just the crucial information, and then people can go further into the build and install docs if needed. In my opinion, a getting started guide should be fairly self contained.

Reply:
Hope I didn't step on your feet too much by removing most of the section you just wrote a couple days earlier... I have now brought back the section about downloading a pre-built binary. That's what most people should be doing anyway.

IMHO the getting started guide is a kind of 'teaser' for various things you can do. Seen in that light, I think the installation part shouldn't be two screens of info with all the detailed options.

Reply:
Why not have a teaser doc and a self-contained doc? We can discuss what to call them and where to put them.

Reply:
@fjl and @adamschmideg In my mind it was the teaser already, I'm finding it somewhat hard to take in all the changes here, so for now I think let this change through, and I'll review it later with what I wrote to see if there's anything I think is particularly missing, and bring it back, perhaps summarised more, or with better jump off points.